exonum/src/blockchain/mod.rs

```diff
@@ -286,6 +286,19 @@ impl Blockchain {
         crypto::hash(&vec)
     }
 
+    #[doc(hidden)]
+    pub fn broadcast_raw_transaction(&self, tx: RawTransaction) -> Result<(), failure::Error> {
+        let service_id = tx.service_id();
```

Comment:
Should we check that `service_id` is valid (i.e., corresponds to an existing service) here?

Reply:
Probably good idea.